src/parser/parser.js

```diff
@@ -87,6 +87,10 @@ const parse = (tokens) => {
                 out += tag("em", true);
             }
 
+            if (token.type === "link") {
+                out += `<a href="${token.href}" title="${token.title}">${token.text}</a>`;
+            }
+
```

Comment:
```suggestion
out += `<a href="${token.href}" title="${token.title}">`
_recursive_parse(token.children);
out += </a>
```

Seamos ambiciosos, parseemos recursivamente el texto que se pone en el link. que pasa si el usuario quiere hacer

```
[***bold italic foo***](http://example.com)
```

Reply:
Estaba confundido en esa parte. Ahora la entiendo mejor 😄 . Hago el cambio propuesto!